pkg/dataflatten/json.go

```diff
@@ -4,13 +4,26 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
+
+	"golang.org/x/text/encoding/unicode"
+	"golang.org/x/text/transform"
 )
 
 func JsonFile(file string, opts ...FlattenOpts) ([]Row, error) {
 	rawdata, err := os.ReadFile(file)
 	if err != nil {
 		return nil, err
 	}
+
+	// Attempt to decode utf16 data.
+	valid_json := json.Valid(rawdata)
+	if !valid_json {
```

Comment:
```suggestion
	if !json.Valid(rawdata) {
	  // Attempt utf16 conversion
```

For this kind of boolean, `ok` is normal, but we can just drop the variable here